src/nft/components/profile/view/ViewMyNftsAsset.tsx

```diff
@@ -0,0 +1,73 @@
+import * as Card from 'nft/components/collection/Card'
+import { AssetMediaType } from 'nft/components/collection/Card'
+import { useBag, useIsMobile, useSellAsset } from 'nft/hooks'
+import { WalletAsset } from 'nft/types'
+import { useMemo, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+const NFT_DETAILS_HREF = (asset: WalletAsset) => `/nfts/asset/${asset.address}/${asset.tokenId}?origin=collection`
```

Comment:
asset.address has been undefined for me so I get a broken link. Also origin=profile